packages/patternfly-4/react-table/src/components/Table/SelectColumn.tsx

```diff
@@ -9,14 +9,13 @@ export interface SelectColumnProps {
 
 export const SelectColumn: React.FunctionComponent<SelectColumnProps> = ({
   children = null as React.ReactNode,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   className = '',
```

Comment:
Is there a reason we need to keep `className` if it's unused?

Reply:
I guess it's because we don't want to allow a `className` to be passed to the `<input>`.

Reply:
If there is a specific reason to pull it out, that's fine. It's just odd to set a default value, so a comment would help.